src/vm/typehashingalgorithms.h

```diff
@@ -0,0 +1,90 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// ---------------------------------------------------------------------------
+// Generic functions to compute the hashcode value of types
+// ---------------------------------------------------------------------------
+
+#pragma once
+#include <stdlib.h>
+
+// DIFFERENT FROM CORERT: Hashing code currently different from CoreRT
+
+//
+// Returns the hashcode value of the 'src' string
+//
+inline static DWORD ComputeNameHashCode(LPCUTF8 src)
```

Comment:
Why did you have to change this to DWORD? I think it is better to keep it as int. It better to avoid Windows specific types in files that potential to be reused between projects.